test/system/TODO.md

```diff
@@ -0,0 +1,112 @@
+testing completes successfully.  For a list of tested platforms, please see [the
+CI configuration file.](../../.cirrus.yml)
+
+
+# Execution
+
+When working from a clone of [the libpod repository](https://github.com/containers/libpod),
+the main entry-point for humans and automation, is `make localsystem`.  When operating
+from a packaged version of the system-tests, the entry-point may vary and therefore
+must be documented for the platform.  Running the packaged system-tests assumes
+a matching version of podman, and all standard dependencies are installed.
+
+
+# Test Design and overview
+
+System-tests should be high-level and user work-flow oriented.  For example, consider
+how multiple podman invocations would be used together by an end-user.  This collection
+of related commands, and the end-result would be considered a single test.  If one or
+more of the intermediate commands fail, the test could still pass as long as the
+end-result is still achieved.
+
+# Matrix:
+
+- As-installed defaults, root user
+- As-installed defaults, regular user
+
+# *TODO*: List of needed System-tests
+
+- [ ] pull, build, run, attach, commit, diff, inspect
+
+  - Pull existing image from registry
+  - Build new image FROM explicitly pulled image
+  - Run built container in detached mode
+  - Attach to running container, execute command to modify storage.
+  - Commit running container to new image w/ changed ENV VAR
+  - Verify attach + commit using diff
+  - verify changed ENV VAR with inspect
+
+- [ ] Implied pull, create, start, exec, log, stop, wait, rm
+
+  - Create non-existing local image
+  - start stopped container
+  - exec simple command in running container
+  - verify exec result with log
+  - wait on running container
+  - stop running container with 2 second timeout
+  - verify wait in 4 seconds or less
+  - verify stopped by rm **without** --force
+
+- [ ] Implied pull, build, export, modify, import, tag, run, kill
+
+  - Build from Dockerfile FROM non-existing local image
+  - Export built container as tarball
+  - Modify tarball contents
+  - Import tarball
+  - Tag imported image
+  - Run imported image to confirm tarball modification, block on non-special signal
+  - Kill can send non-TERM/KILL signal to container to exit
+  - Confirm exit within timeout
+
+- [ ] Container runlabel, exists, checkpoint, exists, restore, stop, prune
+
+  - Using pre-existing remote image, start it with 'podman container runlabel --pull'
+  - Run a named container that exits immediatly
+  - Confirm 'container exists' zero exit (both containers)
+  - Checkpoint the running container
+  - Confirm 'container exists' non-zero exit (runlabel container)
+  - Confirm 'container exists' zero exit (named container)
+  - Run 'container restore'
+  - Confirm 'container exists' zero exit (both containers)
+  - Stop container
+  - Run 'container prune'
+  - Confirm podman ps -a lists no containers
+
```

Comment:
maybe that will come in with the login/logout tests, but I didn't see much authentication kind of tests in the above.

Reply:
Those tests are really hard to do because they're either complex, or they rely heavily on external resources.  We need them, but best cover the basics first.   Both `login` and `logout` are in the TODO list.